stressTesting.js

```diff
@@ -0,0 +1,49 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+export let options = {
+  stages: [
+    { duration: '10s', target: 10 },
+    { duration: '30s', target: 100 },
+    { duration: '30s', target: 300 },
+    { duration: '30s', target: 100 },
+    { duration: '20s', target: 0 },
+  ],
+};
+export default function () {
+  const record = Math.floor(Math.random() * (1000000) + 9000000);
+  if (Math.random() <= 0.1) {
```

Comment:
clever inclusion of the occasional post request :)  Does this average to about 10% of requests? I think we only have to have posts as 5%.

Reply:
@claire-melbourne yes! This does average to 10% but that's good to know about the 5%, I'll probably update for deployment then. :)

Thanks, Claire.